packages/teams-js/test/public/media.spec.ts

```diff
@@ -755,9 +750,11 @@ describe('media', () => {
           errorCode: ErrorCode.INTERNAL_ERROR,
         };
 
-        await new media.VideoController().stop((e: SdkError) => {
-          expect(e).toEqual(err);
-        });
+        try {
+          new media.VideoController().stop(emptyCallback);
+        } catch (error) {
+          expect(error).toEqual(err);
```

Comment:
I think this test should have `expect.assertions(4)` at the top (if my count is right), since we are adding an `expect` in a `catch` block

Reply:
Or more because of the `mobilePlatformMock.initializeWithContext` (which is whole other story, don't get me started...)

Reply:
this is the reason that the expect.assertions() was created, when I added it and the assertions should be 7 because there are 3 outside of this file, the catch was not getting hit

Reply:
Are you saying that when you add `expect.assertions(7)` to the top of this test, the catch does not get hit?